src/vs/workbench/services/suggest/browser/simpleCompletionModel.ts

```diff
@@ -208,10 +208,33 @@ export class SimpleCompletionModel {
 				// Then by file extension length ascending
 				score = a.fileExtLow.length - b.fileExtLow.length;
 			}
-			if (score === 0 || fileExtScore(a.fileExtLow) === 0 && fileExtScore(b.fileExtLow) === 0) {
-				// both files or directories, sort alphabetically
```

Comment:
I probably wrote this comment, but I don't think this is actually true and non-files can enter this block.